scripts/stage_layer.py

```diff
@@ -0,0 +1,94 @@
+from argparse import ArgumentParser
+import logging
+from pathlib import Path
+import shutil
+import subprocess
+import sys
+from zipfile import (
+    ZipFile,
+    ZipInfo,
+)
+
+import boto3
+
+from azul import config
+from azul.logging import configure_script_logging
+
+log = logging.getLogger(__name__)
+
+
+class LayerBuilder:
+    layer_dir = Path(config.project_root) / 'lambdas' / 'layer'
+    out_dir = layer_dir / '.chalice' / 'terraform'
+
+    def __init__(self):
+        self.client = boto3.client('s3')
```

Comment:
```suggestion
        self.s3 = boto3.client('s3')
```

Is (or should be) the convention. `client` doesn't scale to more than one boto3 client instance.